extensions/amp-tiktok/0.1/test/test-amp-tiktok.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-tiktok';
+import {createElementWithAttributes, isAmpElement} from '../../../../src/dom';
+
+describes.realWin(
+  'amp-tiktok',
+  {
+    amp: {
+      extensions: ['amp-tiktok'],
+    },
+  },
+  (env) => {
+    let win;
+    let doc;
+    let clock;
+
+    beforeEach(() => {
+      win = env.win;
+      doc = win.document;
+      clock = env.sandbox.useFakeTimers();
+      clock.tick(0);
+    });
+
+    async function getTiktok(videoId, isFullSrcUrl, opt_locale) {
+      const tiktok = createElementWithAttributes(win.document, 'amp-tiktok', {
+        layout: 'responsive',
+        width: '325px',
+        height: '730px',
+      });
+      if (isFullSrcUrl) {
+        tiktok.setAttribute(
+          'data-src',
+          `https://www.tiktok.com/@scout2015/video/${videoId}?lang=en-US`
```

Comment:
This should not belong in a unit test. This will create the component by fetching the TikTok with the specified videoId, which is what we'd do in an e2e test.

Reply:
@dmanek I would say it's fine. This is already done in almost if not all of the 3p iframe extensions. 

@rebeccanthomas You could do something to load `blank.html` from `localhost` instead, but it's not required.